### PR TITLE
[#867] Add NUCATS tag for Collection

### LINKED ISSUE #867: Add "NUCATS Grants Repository" Tag For All Collection Items
For Collection: [NUCATS Grants Repository](https://digitalhub.northwestern.edu/collections/f2bf6e1d-0e32-4ce2-a52e-bb0522d5708d) 

User request: searching for "human research" returns results from outside "NUCATS Grants Repository" collection. To limit results for items in their collection, adding tag as additional filter.

### PR DESCRIPTION
**Pull request by sharpattack**
For NUCATS Grants Repository collection, add tag to all non collection
items. closes #867

### CODE REVIEW

File: spec/lib/batch/add_tags_for_collection_members_spec.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    nested_collection.update_attributes(members: [nested_item])
+  end
+
+  describe "#add_tags_for_collection_members" do
+    context "for collection with no collections as members" do
+      it "should add the new tag" do
+        expect(item.tag).to eq(["Old Dogs"])
+        expect(collection.tag).to eq(["Old Dogs"])
+
+        add_tag_for_collection_members(collection_id: collection.id, tags: ["NUCATS"])
```
[the remark this answers is not in the document]
Comment (sharpattack) [reply spec/lib/batch/add_tags_for_collection_members_spec.rb:22]: Ah, I could have used "NUCATS Grants Repository" here. The test is actually checking the method `add_tag_for_collection_members`, and you can have whatever tags you want. I should have used some other tag that was not similarly looking to where I am using the method in lib/batch/add_tag_nucats_867.rb above.

File: config/deploy.rb [hunk 2]
```diff
@@ -79,6 +79,7 @@
   SSLCertificateFile #{cert_path}/#{cert_host_name}_cert.cer
   SSLCertificateChainFile #{cert_path}/#{cert_host_name}_interm.cer
   SSLCertificateKeyFile #{cert_path}/#{cert_host_name}_key.cer
+  SSLProtocol all -SSLv2 -SSLv3
```
Comment (Meowcenary) [inline config/deploy.rb:82]: Good job figuring this out. Better security is always great and especially if it's something that informs us for the future too.

**Review by Meowcenary — APPROVED** [review]

**Review by carsonicator — APPROVED** [review]
Thanks, @sharpattack

File: config/deploy.rb (hunk 2 above)
Comment (sharpattack) [reply config/deploy.rb:82]: Thanks. Yea, we had this turned off in other configs, but they didn't matter.

**Pull request merged**